src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -590,12 +590,15 @@ error_code replica_bulk_loader::remove_local_bulk_load_dir(const std::string &bu
 }
 
 // ThreadPool: THREAD_POOL_REPLICATION
-void replica_bulk_loader::cleanup_download_task()
+bool replica_bulk_loader::cleanup_download_task()
 {
```

Comment:
why did you and a return type here?

Reply:
https://github.com/XiaoMi/rdsn/blob/9d1059eb6597efbfb00b405c867d9b0edd82e0ec/src/replica/replica_context.h#L53-L63

CLEANUP_TASK marco will return value, it should only be used in the function whose return value is bool.